mm2src/coins_activation/src/tendermint_with_assets_activation.rs

```diff
@@ -234,6 +236,7 @@ impl PlatformCoinWithTokensActivationOps for TendermintCoin {
         protocol_conf: Self::PlatformProtocolInfo,
     ) -> Result<Self, MmError<Self::ActivationError>> {
         let conf = TendermintConf::try_from_json(&ticker, coin_conf)?;
+        let is_keplr_from_ledger = activation_request.is_keplr_from_ledger && activation_request.with_pubkey.is_some();
```

Comment:
Q: Can this be a `keplr from ledger` but not `with pubkey`?

Reply:
No, it can't. If you are using hardware wallet, `with-pubkey` is the only way you can use your wallet with mm2.

Reply:
Aha, then does it make sense to `&& activation_request.with_pubkey.is_some()`. If we suspect the activation request might be malformed, maybe we could error here instead.